### DISCUSSION TITLE
How to include original containers of selected files in report

### DISCUSSION THREAD

**Discussion by janieltonveloso** [sepinf-inc/IPED#110] [Q&A] [answered]
how to generate a report of files and their containers to appear in an official report ?

**Comment by lfcnassif** [comment, marked as answer]
There is no sugar button for that currently. But you can display parentIds column in table view and export properties of selected files. Open the CSV file created, copy parentIds column content and do the following search:
`hasSubitem:true AND id:(parentId1 parentId2 parentId3...)`

That should return all containers which sub items were in your original list, so you can add them to a bookmark and report.

**Comment by lfcnassif** [comment]
Did it work?

**Comment by janieltonveloso** [comment]
I understood, I thought it had a simpler form. In my case there are a lot
of files with containers and some are relevant but these are still a lot.
The solution will be to select a sample and show its containers. The great
triumph is to show that we find steganography.

thanks!!

Em qua., 3 de jun. de 2020 às 09:12, Luis Nassif <notifications@github.com>
escreveu:

> There is no sugar button for that currently. But you can display parentIds
> column and export properties of relevant files. Open the csv file created,
> copy parentIds column content and do the following search:
> hasSubitem:true AND id:(parentId1 parentId2 parentId3...)
> That should return all containers which sub items were in your original
> list, so you can add them to report.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lfcnassif/IPED/issues/110#issuecomment-638156509>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHDK2VAMK7XPHELYSKY4OPTRUY42ZANCNFSM4NRF6OYQ>
> .
>


-- 
[Janielton de Sousa Veloso]
